src/ui/interaction.rs

```diff
@@ -65,8 +65,12 @@ fn trigger_interaction_sfx(
 ) {
     for interaction in &interactions {
         match interaction {
-            Interaction::Hovered => commands.trigger(PlaySfx::Key(SfxKey::ButtonHover)),
-            Interaction::Pressed => commands.trigger(PlaySfx::Key(SfxKey::ButtonPress)),
+            Interaction::Hovered => {
+                commands.play_sound_effect(SoundEffectHandles::KEY_BUTTON_HOVER)
```

Comment:
I think it's interesting that we defined an extension trait and this is still more verbose than it was. Mostly due to spelling out `sound_effect` every time instead of `sfx`. Not necessarily a suggestion, just a remark :p

Reply:
That's true, hehe.